publish-package.sh

```diff
@@ -5,20 +5,20 @@ set -e
 CURRENT_BRANCH=`git name-rev --name-only HEAD`
 
 if [ $CURRENT_BRANCH != 'master' ] ; then
```

Comment:
hmm, actually, the way to detect the current branch should work, as PR builds won't be on the commit on the master branch. You are not using the Travis env variable to detect the branch